src/main/resources/site/error/error.html

```diff
@@ -6,15 +6,15 @@
     <title>Superhero theme</title>
 
     <!--[if lt IE 9]>
-    <script data-th-src="${portal.assetUrl({'_path=js/html5.js'})}" src="../assets/js/html5.js" type="text/javascript"></script>
+    <script data-th-src="${assetUrlPrefix + '/html5.js'}" src="../assets/js/html5.js" type="text/javascript"></script>
```

Comment:
```suggestion
    <script data-th-src="${assetUrlPrefix}/html5.js" src="../assets/js/html5.js" type="text/javascript"></script>
```

Reply:
Why we use `src="../assets/js/html5.js"`? I think it will be broken. The same thing for the rest of parts.